### ISSUE TITLE
Why can't I use `attrs` on a custom tag?

### ISSUE THREAD

**Issue by kristianmandrup** [marko-js/marko#135]
I have the following template

```html
<div class="ui list">
  <for each="item in data.list">
    $item.ui
    <icon-item attrs="item">
      <div tag-body="item.body" class="description"></div>
    </icon-item>
  </for>
</div>
```

with the following data"

```js
[ { ui: 'large orange',
    icon: 'user',
    label: 'kmandrup',
    body: 'I DREAM in CODE' },
  { ui: 'large blue',
    icon: 'user',
    label: 'isaura',
    body: 'I LOVE web DESIGN' } ]
```

It correctly prints `large orange` and so on for `$item.ui` but for `<icon-item attrs="item"` no attributes are passed to the custom tag `icon-item`. Why is that?

In the docs for `attrs` it is used on a normal `div` tag, but I would have assumed I could also use this technique for custom tags??? Otherwise tag maintenance will quickly become very "fragile" IMO.

Even more convenient would be if I could somehow list the attributes I wanna transfer, then have marko use the schema to transfer and type convert values accordingly:

`use-attrs="item:name,icon,label"`

**Comment by patrick-steele-idem** [comment]
`attrs` currently only applies HTML tags (not custom tags). You can, however, do the following:

```xml
<icon-item c-data="item">
```

However, now that you point it out. We should allow: 

```xml
<icon-item attrs="item">
```

I will make that change.

**Comment by kristianmandrup** [comment]
Sweet :)

**Comment by patrick-steele-idem** [comment]
Fixed. New version published: `marko@2.7.17`
